tests/http/src/test/java/org/mule/test/module/http/functional/requester/HttpRestrictedCiphersAndProtocolsTestCase.java

```diff
@@ -46,7 +53,9 @@
   @Rule
   public ExpectedError expectedError = ExpectedError.none();
 
-  private HttpRequestOptionsBuilder optionsBuilder = HttpRequestOptionsBuilder.newOptions().method(POST.name());
+  // Create a new HttpClient because i need to configure the TLS context per test
```

Comment:
Create a new HttpClient because i need to configure the TLS context per test > Creates a new HttpClient because is needed to configure the TLS context per test